cmd/sqlcmd/sqlcmd.go

```diff
@@ -228,6 +228,7 @@ func Execute(version string) {
 			setVars(vars, &args)
 
 			if args.Version {
+				fmt.Println(localizer.Sprintf("sqlcmd: Install/Create/Query SQL Server, Azure SQL, and Tools"))
```

Comment:
>ocalizer.Sprintf("sqlcmd: Install/Create/Query SQL Server, Azure SQL, and Tools" [](http://example.com/codeflow?start=17&length=80)

2 things, for localizability
1 - declare the string in a higher scope so there's only 1 copy of it.
2 - run `build\build` on your Windows enlistment to generate the localization files to include in the PR

Without 1 I think the loc tools will have an error about a duplicate key.

Reply:
@shueybubbles I'm not a Go developer by any stretch of the imagination, and this edit was done in-browser. I have no local environment to make this change.

Reply:
Pushed above changes to branch